src_c/event.c

```diff
@@ -573,6 +603,12 @@ dict_from_event(SDL_Event *event)
             _pg_insobj(dict, "pinched", PyFloat_FromDouble(event->mgesture.dDist));
             _pg_insobj(dict, "num_fingers", PyInt_FromLong(event->mgesture.numFingers));
             break;
+        case SDL_MOUSEWHEEL:
+            /* https://wiki.libsdl.org/SDL_MouseWheelEvent */
+            obj = Py_BuildValue("(ii)", event->wheel.x, event->wheel.y);
+            _pg_insobj(dict, "pos", obj);
```

Comment:
`pos` is actually the amount scrolled, not the position. It should probably have a different name